lib/encoding/mnemonic.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Elixium.Mnemonic do
```

Comment:
It looks like some of the functions in here are already defined in the Elixium.Advanced module. We should probably merge the two